src/libcore/intrinsics.rs

```diff
@@ -609,3 +595,14 @@ extern "rust-intrinsic" {
     /// is thrown (aka the thread panics).
     pub fn try(f: fn(*mut u8), data: *mut u8) -> *mut u8;
 }
+
+#[cfg(not(stage0))]
+extern "rust-intrinsic" {
+    /// Performs an unchecked division, resulting in undefined behavior
+    /// where y = 0 or x = `T::min_value()` and y = -1
+    pub fn unchecked_div<T>(x: T, y: T) -> T;
```

Comment:
I think these can be in the same place they were in above, the `#[cfg]` should work ok on the functions as well as the block itself.